_scripts/ci/bintray-template.json

```diff
@@ -1,6 +1,6 @@
 {
   "package": {
-    "name": "helm",
+    "name": "helmc",
     "repo": "helm-ci",
```

Comment:
i had created the helmc package(https://bintray.com/deis/helm-ci/helmc/view) in the bintray....not sure if the travis automatically creates it